soh/soh/SaveManager.cpp

```diff
@@ -932,6 +935,10 @@ void SaveManager::SaveSection(int fileNum, int sectionID, bool threaded) {
     if (fileNum == 0xFF || fileNum == 0xFE) {
         return;
     }
+    // Don't save if after OnExitGame
```

Comment:
nitpick: this comment feels a little odd to me

```suggestion
    // Make sure we have something to save
```

or just no comment (not being able to save without a save in memory feels intuitive) would be my preference

Reply:
But it's not just about having something to save. The saves typically would still end up with valid copies of SaveContext, and nothing triggers a save before loading in, so it's literally just about preventing saves after reset/exit. What about "Don't save outside of a loaded game"?